.changeset/heavy-seahorses-poke.md

```diff
@@ -0,0 +1,20 @@
+---
+'astro': minor
+---
+
+The Content Layer API introduced behind a flag in [4.14.0](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md#4140) is now stable and ready for use.
+
+The Content Layer API is the next generation of content collections, allowing you to load content from anwhere. For more details, see the [content collections documentation](https://docs.astro.build/en/guides/content-collections/).
```

Comment:
I will also note in general, it's is OK to hype the feature here! Many people do not try features until they are stable. 

You can see this changeset for unflagging `astro:env` as an example, and my comments there. (You can pull from the original experimental changeset, or from existing config references, if you want!) #11679 

PS - ignore the link used in this changeset. I edited it before the CHANGELOG was published so that it wouldn't be a broken link

Reply:
I've updated it, largely based on the experimental changeset